src/systemd/sd-lldp.h

```diff
@@ -33,6 +33,87 @@ _SD_BEGIN_DECLARATIONS;
 typedef struct sd_lldp sd_lldp;
 typedef struct sd_lldp_neighbor sd_lldp_neighbor;
 
+#define SD_LLDP_MULTICAST_ADDR     { 0x01, 0x80, 0xc2, 0x00, 0x00, 0x0e }
```

Comment:
I think it falls into the category of "not strictly necessary, but harmless and useful to users". Anyone using `sd-lldp.h` is likely to want this.